subgraphs/uniswap-v3/src/common/utils/utils.ts

```diff
@@ -286,3 +287,17 @@ export function stringToBytesList(list: string[]): Bytes[] {
   }
   return result;
 }
+
+export function absBigInt(value: BigInt): BigInt {
```

Comment:
there is actually a built in abs() func for BigInt, it doesn't look like that is the case for BD though

Reply:
Oh right. I removed it.